modules/ob-creating-a-buildah-build.adoc

```diff
@@ -39,20 +42,38 @@ spec:
     image: image-registry.openshift-image-registry.svc:5000/buildah-example/sample-go-app
 EOF
 ----
++
+.Example: Using `shp` CLI
+[source,terminal]
+----
+$ shp build create buildah-golang-build \
+--source-url="https://github.com/shipwright-io/sample-go" --source-context-dir="docker-build" \ <1>
+--strategy-name="buildah" \ <2>
+--dockerfile="Dockerfile" \ <3>
+--output-image="image-registry.openshift-image-registry.svc:5000/buildah-example/go-app" <4>
```

Comment:
```suggestion
$ shp build create buildah-golang-build \
--source-url="https://github.com/shipwright-io/sample-go" --source-context-dir="docker-build" \// <1>
--strategy-name="buildah" \// <2>
--dockerfile="Dockerfile" \// <3>
--output-image="image-registry.openshift-image-registry.svc:5000/buildah-example/go-app" <4>
```
These extra slashes are needed to fix syntax highlighting in multi-line code blocks that contain callouts. See https://github.com/openshift/openshift-docs/blob/main/modules/installation-azure-preparing-diskencryptionsets.adoc?plain=1#L26 for another example of this syntax